<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.8.14">
  <compounddef id="classfunction__wrapper_1_1_n_y_i_error" kind="class" language="Python" prot="public">
    <compoundname>function_wrapper::NYIError</compoundname>
    <basecompoundref prot="public" virt="non-virtual">Exception</basecompoundref>
      <sectiondef kind="public-attrib">
      <memberdef kind="variable" id="classfunction__wrapper_1_1_n_y_i_error_1af98bd7b4fcf527a054e00dd9d4a1688b" prot="public" static="no" mutable="no">
        <type></type>
        <definition>function_wrapper.NYIError::reason</definition>
        <argsstring></argsstring>
        <name>reason</name>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="/Users/robkunkle/fork/goodlux/pytorch/aten/src/ATen/function_wrapper.py" line="200" column="1" bodyfile="/Users/robkunkle/fork/goodlux/pytorch/aten/src/ATen/function_wrapper.py" bodystart="200" bodyend="-1"/>
      </memberdef>
      </sectiondef>
      <sectiondef kind="public-func">
      <memberdef kind="function" id="classfunction__wrapper_1_1_n_y_i_error_1a29783c73954cff4e5bb80918f433ee12" prot="public" static="no" const="no" explicit="no" inline="no" virt="non-virtual">
        <type>def</type>
        <definition>def function_wrapper.NYIError.__init__</definition>
        <argsstring>(self, reason)</argsstring>
        <name>__init__</name>
        <param>
          <type>self</type>
          <defname>self</defname>
        </param>
        <param>
          <type>reason</type>
          <defname>reason</defname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="/Users/robkunkle/fork/goodlux/pytorch/aten/src/ATen/function_wrapper.py" line="199" column="1" bodyfile="/Users/robkunkle/fork/goodlux/pytorch/aten/src/ATen/function_wrapper.py" bodystart="199" bodyend="202"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
<para><verbatim>Indicates we don&apos;t support this declaration yet</verbatim> </para>    </detaileddescription>
    <inheritancegraph>
      <node id="462">
        <label>function_wrapper.NYIError</label>
        <link refid="classfunction__wrapper_1_1_n_y_i_error"/>
        <childnode refid="463" relation="public-inheritance">
        </childnode>
      </node>
      <node id="463">
        <label>Exception</label>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="464">
        <label>function_wrapper.NYIError</label>
        <link refid="classfunction__wrapper_1_1_n_y_i_error"/>
        <childnode refid="465" relation="public-inheritance">
        </childnode>
      </node>
      <node id="465">
        <label>Exception</label>
      </node>
    </collaborationgraph>
    <location file="/Users/robkunkle/fork/goodlux/pytorch/aten/src/ATen/function_wrapper.py" line="196" column="1" bodyfile="/Users/robkunkle/fork/goodlux/pytorch/aten/src/ATen/function_wrapper.py" bodystart="196" bodyend="203"/>
    <listofallmembers>
      <member refid="classfunction__wrapper_1_1_n_y_i_error_1a29783c73954cff4e5bb80918f433ee12" prot="public" virt="non-virtual"><scope>function_wrapper::NYIError</scope><name>__init__</name></member>
      <member refid="classfunction__wrapper_1_1_n_y_i_error_1af98bd7b4fcf527a054e00dd9d4a1688b" prot="public" virt="non-virtual"><scope>function_wrapper::NYIError</scope><name>reason</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
